spec/bundle/dumper_spec.rb

```diff
@@ -16,8 +16,8 @@
     Bundle::WhalebrewDumper.reset!
     Bundle::BrewServices.reset!
 
-    chrome = instance_double("Cask::Cask", full_name: "google-chrome")
-    java = instance_double("Cask::Cask", full_name: "java")
+    chrome = instance_double("Cask::Cask", to_s: "google-chrome")
+    java = instance_double("Cask::Cask", to_s: "java")
```

Comment:
As we identified a regression from a previous change, I'd think we would want to add a test case covering that scenario. Thoughts?

Reply:
Seems like a good idea. Would that involve adding a double that's a cask from a tap? I'll think about how to implement that.

Reply:
I added a test case but I don't think it's actually useful. Could I have some advice on a good way to test this?

Reply:
Which part don't you don't is useful?